manifests/t/TorGuard/TorGuard/latest/TorGuard.TorGuard.yaml

```diff
@@ -2,16 +2,16 @@ PackageIdentifier: TorGuard.TorGuard
 PackageVersion: latest
 PackageName: TorGuard
 Publisher: TorGuard
-License: Copyright Â© 2020 TorGuard.net
+License: Copyright © 2020 TorGuard.net
 Moniker: torguard
 Tags:
-- vpn openvpn
+- vpn openvpn wireguard
```

Comment:
These should be an array like:
 \- vpn
 \- openvpn
 \- wireguard

Reply:
Ok, will fix. That's something new I didn't know.